css/css-box/margin-trim/block-container-non-adjoining-item-ref.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>CSS margin-trim: block-container-non-adjoining-item</title>
+<link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-block">
+<meta name="assert" content="non-adjoining first child should still have its margins trimmed and the container should keep its set margin">
+<style>
+.container {
+    background-color: green;
+    width: 100px;
+    margin-bottom: 100px;
+    padding-bottom: 50px;
+    background-clip: padding-box;
+}
+.outer {
+    background-color: green;
+    width: 100px;
+    height: 50px;
+}
+.child {
+    height: 50px;
+}
+</style>
+</head>
+<body>
+<div>
+<div class="container">
+    <div class="child"></div>
+</div>
```

Comment:
The classless wrapper-`<div>` (line 27, closed at 32) seems unnecessary and removable as well.

(This also applies to the matching testcase-file.)